src/pages/_layout.svelte

```diff
@@ -7,10 +8,11 @@
 <Footer items={items} socials={socials} contact={contact}/>
 
 <script>
-  import items from '@/assets/navigation.js'
-  import socials from '@/assets/socials.js'
-  import contact from '@/assets/contact.js'
+    import items from '@/assets/navigation.js'
```

Comment:
Why did you add these indents here?

Reply:
My IDE do things like this. Do you have custom style for svelte (your style)?

Reply:
Hm, I think I have just a Svelte plugin: https://plugins.jetbrains.com/plugin/12375-svelte

Reply:
I think, now all is okay :)